### ISSUE TITLE
Uploaded Status Cancelled only on iOS 

### ISSUE THREAD

**Issue by DavidAriza** [fluttercommunity/flutter_uploader#234]
I'm trying to upload a file on iOS but it's not working, I get statusCode 500 and Upload Status 5, any idea why is this happening only on iOS? In Android is working fine

**Comment by jaysignorello** [comment]
I'm having the same problem. However, oddly it works for me in debug mode on a real device and in the simulator. When I make a release build, it stops working and I don't receive any result events, though it enqueues the tasks without any problems. The same web url, data, and file uploads fine in flutter.

**Comment by waqadArshad** [comment]
@jaysignorello did you find any solution? Please help me out here. The app is live and this is unbearable. Thanks

**Comment by jaysignorello** [comment]
@waqadArshad yes, I setup the listener to work in the foreground, which seems to work fine while the user is using the app, but certainly isn't ideal given it's not working as intended.
